src/julia/JuliaTypes.jl

```diff
@@ -92,7 +92,8 @@ end
 #
 ###############################################################################
 
-const RingElement = Union{RingElem, Integer, Rational, AbstractFloat}
+const RingElement   = Union{RingElem,   Integer, Rational, AbstractFloat}
+const NCRingElement = Union{NCRingElem, Integer, Rational, AbstractFloat}
```

Comment:
If you are going to do this then everything in RingElement should be in there, since any commutative ring is not necessarily commutative (which is what NCRing is all about).